src/components/Article.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Article({
+  title,
+  text,
+  date,
+}) {
+  <h1>{title}</h1>;
+    <p>{text}</p>;
+      <span>{date}</span>;
```

Comment:
```suggestion
  <h1>{title}</h1>;
  <p>{text}</p>;
  <span>{date}</span>;
```
Навіщо тут ця "драбинка"?